src/wp-content/themes/twentyfifteen/functions.php

```diff
@@ -412,7 +412,13 @@ function twentyfifteen_fonts_url() {
  * @since Twenty Fifteen 1.1
  */
 function twentyfifteen_javascript_detection() {
-	echo "<script>(function(html){html.className = html.className.replace(/\bno-js\b/,'js')})(document.documentElement);</script>\n";
+	$js = "(function(html){html.className = html.className.replace(/\bno-js\b/,'js')})(document.documentElement);";
+
+	if ( function_exists( 'wp_print_inline_script_tag' ) ) {
+		wp_print_inline_script_tag( $js . "\n//# sourceURL=" . __FUNCTION__ );
```

Comment:
Per @sirreal in https://github.com/WordPress/wordpress-develop/pull/9955#pullrequestreview-3258575289

```suggestion
		wp_print_inline_script_tag( $js . "\n//# sourceURL=" . rawurlencode( __FUNCTION__ ) );
```